### PR TITLE
add the license file to the deb package

### PR DESCRIPTION
**Pull request by delgod**
It is very important (from the legal point of view) to distribute the license together with binaries.

### CODE REVIEW

**Comment by defbin** [comment]
Thank you 😸 LGTM. Let's wait for @EvgeniyPatlan's approval

**Comment by dAdAbird** [comment]
Should we do for the RPM the same?

**Review by dAdAbird — APPROVED** [review]

**Review by defbin — APPROVED** [review]

**Pull request merged**
